src/test/scala/uk/gov/nationalarchives/draftmetadatavalidator/LambdaSpec.scala

```diff
@@ -24,41 +24,103 @@ class LambdaSpec extends ExternalServicesSpec {
     )
   }
 
-  def mockS3PutResponse(): StubMapping = {
+  def mockS3ErrorFilePutResponse(): StubMapping = {
     wiremockS3.stubFor(
-      put(urlEqualTo(s"/$consignmentId/sample.csv"))
+      put(urlEqualTo(s"/$consignmentId/draft-metadata-errors.json"))
         .willReturn(aResponse().withStatus(200))
     )
   }
 
-  "handleRequest" should "download the draft metadata csv file, validate and save to db if it has no errors" in {
+  "handleRequest" should "download the draft metadata csv file, validate, save error file to s3 and save to metadata to db if it has no errors" in {
     authOkJson()
     graphqlOkJson(true)
     mockS3GetResponse("sample.csv")
+    mockS3ErrorFilePutResponse()
     val input = Map("consignmentId" -> consignmentId).asJava
     val response = new Lambda().handleRequest(input, mockContext)
     response.getStatusCode should equal(200)
+
+    val s3Interactions: Iterable[ServeEvent] = wiremockS3.getAllServeEvents.asScala.filter(serveEvent => serveEvent.getRequest.getMethod == RequestMethod.PUT).toList
+    s3Interactions.size shouldBe 1
+
+    val errorWriteRequest = s3Interactions.head
+    val errorFileData = errorWriteRequest.getRequest.getBodyAsString
+    val today = org.joda.time.DateTime.now().toString("yyyy-MM-dd")
+    val expectedErrorData = s"""{
+                               |  "consignmentId" : "f82af3bf-b742-454c-9771-bfd6c5eae749",
+                               |  "date" : "$today",
+                               |  "validationErrors" : [
+                               |  ]
+                               |}""".stripMargin
+    errorFileData shouldBe expectedErrorData
   }
 
-  "handleRequest" should "download the draft metadata csv file, validate it and re-upload to s3 bucket if it has any errors" in {
+  "handleRequest" should "download the draft metadata csv file, validate it and save error file to s3" in {
```

Comment:
I would re-word this slightly: `... validate it and save error file with errors to s3"`